made mutation edit button x-large to make it easier to click.

wxtim opened the pull request:
May close #1096 

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Already covered by existing tests.
- [x] No change log entry required (v small change)
- [x] No documentation update required.

MetRonnie reviewed (changes requested):
Icon is now outsized

![image](https://user-images.githubusercontent.com/61982285/184618979-43634da9-e483-4175-8512-450398db45d8.png)

I have opened https://github.com/wxtim/cylc-ui/pull/4 against this with some improvements

MetRonnie commented on src/styles/cylc/_menu.scss:
```diff
@@ -38,4 +38,7 @@
       line-height: 1.35;
     }
   }
+  .pad-button {
+    padding: 30px;
+  }
```
There is no element with the class `pad-button` so this does nothing

wxtim replied:
I did not want to commit this file - this was something I played with and discarded.

wxtim commented:
@MetRonnie  I've removed the `icon` property from the button so it goes back to being square, and I've added an outline so it's completely clear what is and is not the button. I think either of these measures would be sufficient on their own, but in combination...

MetRonnie commented:
Not sure I like the rectangular button or outline. IMHO draws too much attention for a secondary control.

![image](https://user-images.githubusercontent.com/61982285/184843418-232090a3-f8e5-4c6f-95d0-76e96c16f6c6.png)

My proposal:

![image](https://user-images.githubusercontent.com/61982285/184843556-1932ed5c-99dd-414e-92b6-6800aa08f5d9.png)

wxtim commented:
> Not sure I like the rectangular button or outline. IMHO draws too much attention for a secondary control.
> 
> ![image](https://user-images.githubusercontent.com/61982285/184843418-232090a3-f8e5-4c6f-95d0-76e96c16f6c6.png)
> 
> My proposal (reverts [f079595](https://github.com/cylc/cylc-ui/commit/f079595e5742a7f6af8540b10fcb7ec0f7c1a95c)):
> 
> ![image](https://user-images.githubusercontent.com/61982285/184843556-1932ed5c-99dd-414e-92b6-6800aa08f5d9.png)

How about the rectangular button without the edge? 

I don't think anyone will move their mouse over there if they don't want the edit button, and pressing the edit button wrongly is the fail-safe way...

hjoliver commented:
I think the subtly-shaded round button looks best.  Is @MetRonnie's proposal still bigger/more-clickable than the status quo?

wxtim commented:
> I think the subtly-shaded round button looks classiest. Is @MetRonnie's proposal bigger/more-clickable than the status quo? [Looking back at the issue, it is bigger - is that not enough in your view @wxtim ?]

Whatever, I took this on as a learning experience, I'm not militant - If the consensus is the round button, I'll do the round button.
